set.mm

```diff
@@ -40170,7 +40170,7 @@ will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
       AVBUQUODCIZJZVAVBUNUPVHUOCDTUAVAVIVBVAVIDBIZVHJVBVAUOVJVHADBNOBCDFGPQRUGS
       UHUIUTUQVCABCDUJVCUOVHUPABDCUJDCTUKULUM $.
 
-    $( Obsolete as of 9-Sep-2021.  Use ~ prel12g instead.  (Contributed by NM,
+    $( Obsolete as of 16-Jun-2022.  Use ~ prel12g instead.  (Contributed by NM,
```

Comment:
prel12OLD is referenced in dfac2OLD, so it cannot be deleted before that OLD theorem.

Reply:
Yes, therefore I increased the due date so that it can be removed together with the OLD theorems using it.

Reply:
My comment was meant to help understanding why the deletion time was postponed.  This cannot be understood by looking just at the piece of text presented here.  In fact, it is used in prel12gOLD, too, but that goes out of scope earlier.